.github/workflows/publish.yml

```diff
@@ -27,71 +24,10 @@ jobs:
         with:
           github_token: ${{ github.token }}
 
-      - name: Decrypt & Import OSBotify GPG key
-        run: |
-          cd .github
-          gpg --quiet --batch --yes --decrypt --passphrase="$LARGE_SECRET_PASSPHRASE" --output OSBotify-private-key.asc OSBotify-private-key.asc.gpg
-          gpg --import OSBotify-private-key.asc
-        env:
-          LARGE_SECRET_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
-
-      - name: Set up git for OSBotify
-        run: |
-          git config --global user.signingkey AEE1036472A782AB
-          git config --global commit.gpgsign true
-          git config --global user.name OSBotify
-          git config --global user.email infra+osbotify@expensify.com
-
-      - uses: actions/setup-node@v3
+      - uses: Expensify/GitHub-Actions/.github/workflows/npmPublish.yml@main
```

Comment:
FYI I wonder if we should create versions for this whenever we make code changes in this repo.

Reply:
also for testing you can pin it to your branch I think?

Reply:
Once I confirm it works, we can tag that commit as 1.0 and update all `main` references to that 👍